docs/file-reference/index.md

```diff
@@ -297,6 +319,59 @@ projects:
 | spareCheckoutDir | string | What directory to use when pulling |
 
 
+# StarterProject Object
+
+> Pulling from a git location
+
+```yaml
+starterProjects:
+  - name: quarkus-quickstart
+    description: minimal CRUD app with quarkus
+    git:
+      location: https://github.com/quarkusio/quarkus-quickstarts
+      branch: master
+      sparseCheckoutDir: /getting-started/ # optional checkout dir
+      startPoint: 1.7.0.Final # tag or commit to start at
+```   
+
+
+> Pulling from a GitHub location
+
+```yaml
+projects:
```

Comment:
if we removed `projects` shouldn't we remove these parts too? Bit confused. Thought we were removing `projects` too haha

Reply:
removed projects